### PR TITLE
Godwoken syndrome will now stop functioning when RNG gets stuck instead of spamming.

### LINKED ISSUE #55392: Voice of god nearing spam filter
Reporting client version: 513.1536

## Round ID:
[151710](https://scrubby.melonmesa.com/round/151710)

## Testmerges:
Testmerge is not the cause of the issue

## Reproduction:

I got voice of god once from a lobotomy, from bwoinks the admin said that voice of god was repeating "YOU. STOP." and that was nearing the spam filter, not sure if i got muted by the spam limit

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Fixes #55392.

## Why It's Good For The Game
Bugfix

## Changelog
:cl:
fix: Eldritch gods no longer speak fast enough to trigger spam filters.
/:cl:

### CODE REVIEW

**Comment by XDTM** [comment]
I swear byond's rng has a tendency to get stuck at times, the kind of situation described in the bug feels way too frequent when i play or test the game

**Comment by interestingusernam3** [comment]
Yeah this bug is spam of "YOU. STOP", so the RNG is clearly stuck on some number that  picks that phrase (out of that list, it's absolutely not special at all). This just makes it so the bug happens rarely (I hope). If there is some way to change BYOND RND from within this repo this should be closed as such a fix would clearly be prefferable.

**Comment by XDTM** [comment]
not only that, but rolling a 4% chance with enough frequency to trigger the spam alert, on top of the chance of rolling the same phrase every time

**Comment by ATH1909** [comment]
Surely it'd be better to find the source of this problem and fix that, instead of just patching the symptoms?

**Comment by interestingusernam3** [comment]
>Surely it'd be better to find the source of this problem and fix that, instead of just papering over the symptoms?
I now made it call RNG twice and check if it's equal to two different values. The chance for success is still 4% but now it will never succeed when the  RNG gets stuck.

**Comment by XDTM** [comment]
I don't know if using rand would change much, if the RNG engine does get stuck like i think, it'll keep generating the same result no matter the method. It's probably worth experimenting with, but this spot is a bit too niche to do so.

Honestly i'd probably reset it back to how it was and create a separate issue report for the RNG oddities, closing this one as part of it. If we do find a fix it will have to be used wherever we use probability, so it'll have to be a large refactor either way.

**Comment by interestingusernam3** [comment]
>I don't know if using rand would change much, if the RNG engine does get stuck like i think, it'll keep generating the same result no matter the method. It's probably worth experimenting with, but this spot is a bit too niche to do so.

That's why this fixes it. It first checks if the number is 2, then if it's 3. If it's stuck it will always fail. 

>Honestly i'd probably reset it back to how it was and create a separate issue report for the RNG oddities, closing this one as part of it. If we do find a fix it will have to be used wherever we use probability, so it'll have to be a large refactor either way.

This is a problem with BYOND and will therefoere probably never be fixed.

File: code/datums/brain_damage/special.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 /datum/brain_trauma/special/godwoken/on_life()
 	..()
-	if(prob(4))
+	if((rand(1,5)==2)&&rand(1,5==3)) //This check should fail if the RNG is stuck
```
Comment (Jared-Fogle) [inline code/datums/brain_damage/special.dm:15]: 5 == 3...? did you test this dude

**Comment by Jared-Fogle** [comment]
what the fuck is this

**Review by Mothblocks — CHANGES_REQUESTED** [review]
this doesn't fix the bug, it's just stupid. make random words of god not trigger the spam filter.

**Pull request closed without merging**